geode-core/src/main/java/org/apache/geode/management/internal/cli/i18n/CliStrings.java

```diff
@@ -2244,6 +2244,9 @@
   public static final String START_GATEWAYSENDER__MEMBER__HELP =
       "Name/Id of the member on which to start the Gateway Sender.";
 
+  public static final String START_GATEWAYSENDER__CLEAN_QUEUE = "cleanQueues";
+  public static final String START_GATEWAYSENDER__CLEAN_QUEUE__HELP =
+      "Option to clean existing queue at start of the Gateway Sender.";
```

Comment:
Remember to add this option to the documentation of `start gateway-sender` command: https://geode.apache.org/docs/guide/110/tools_modules/gfsh/command-pages/start.html#topic_AB8BA3F42B9645A8BE9BD97CE2F839A8

(`geode/geode-docs/tools_modules/gfsh/command-pages/start.html.md.erb` file)

Reply:
Thanks

Reply:
@mivanac Choice of camel-case for the gfsh option name (--cleanQueues) deviates from past convention. Camel-case is unreliable given different case-sensitivity conventions in different environments. Past practice is to delineate words using a hyphen. I strongly recommend changing the name to `--clean-queues` in accord with past practice.
Note: I also added a 'docs' component to the JIRA ticket, as docs are affected by this PR.

Reply:
@mivinac Your recent commit addressed my issue with the gfsh option name. Thank you!